File: RelationExample/RelationExample/GraphicCard.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace RelationExample
{
    class GraphicCard
    {
        private int dRam;

        public int DRam { get => dRam; set => dRam = value; }

        public GraphicCard(int dRam)
        {
            this.dRam = dRam;
        }
    }
}

File: RelationExampleC++/RelationExampleC++/GraphicCard.cpp
#include "GraphicCard.h"


GraphicCard::GraphicCard(int dRam) : m_dRam(dRam)
{}

int GraphicCard::GetDRam()
{
	return m_dRam;
}

void GraphicCard::SetDRam(int dRam)
{
	m_dRam = dRam;
}